docs/shared-content-source/docs/modules/develop/pages/monitoring.adoc

```diff
@@ -0,0 +1,100 @@
+:page-partial:
+
+include::ROOT:partial$include.adoc[]
+
+## Monitoring Akka Streamlets
+
+Cloudflow allows you to use https://developer.lightbend.com/docs/telemetry/current/home.html[telemery] just by adding the necessary configuration. Telemetry provides instrumentation to report metrics for Akka components through different plugins, such as Prometheus or Elasticsearch. These metrics can then be accessed through, for instance, a Graphana dashboard or a Kibana one. 
+
+
+In this case, we'll describe the use case where the reporter is Prometheus and the dashboard is Grafana. Which is the same as in the https://developer.lightbend.com/docs/cloudflow/current/install/install-console.html[Cloudflow Console]
+
+The required configuration is, as per https://developer.lightbend.com/docs/telemetry/current/setup/cinnamon-agent-sbt.html#lightbend-telemetry-sbt-plugin[documentation], the following:
+
+### Add sbt-cinnamon plugin 
+We must add sbt-cinnamon plugin into `plugins.sbt` file, in the `project` folder.  
+
+```
+addSbtPlugin("com.lightbend.cinnamon" % "sbt-cinnamon" % [version])
+```
+changing `version` to https://developer.lightbend.com/docs/telemetry/current/setup/cinnamon-agent-sbt.html#lightbend-telemetry-sbt-plugin[latest]
+
+### Add library dependencies
+
+The required libraries that will be in charge of adding the telemetry for each Akka component plus the reporter of choice. In our sample
+
+```Cinnamon.library.cinnamonAgent,
+Cinnamon.library.cinnamonAkka,
+Cinnamon.library.cinnamonAkkaStream,
+Cinnamon.library.cinnamonAkkaHttp,
+Cinnamon.library.cinnamonPrometheus,
+Cinnamon.library.cinnamonPrometheusHttpServer
+```
+
+The libraries `cinnamonAkka`, `cinnamonAkkaStream`, and `cinnamonAkkaHttp` would be in charge of adding the metrics, also called https://developer.lightbend.com/docs/telemetry/current/instrumentations/instrumentations.html#instrumentations[instrumentations].  
+`cinnamonPrometheus` and `cinnamonPrometheusServer` are the ones providing the reporter side. For more info about this have a look at https://developer.lightbend.com/docs/telemetry/current/plugins/prometheus/prometheus.html[backend plugins]. Finally, `cinnamonAgent` works under the hood to add the `cinnamon-agent-[version].jar` library that we have to point to when deploying the app. This last point is further explained at the end of this section, on "Add --conf"
```

Comment:
Just say that we are going to refer to the agent jar later and that we need it to be available in the pod.

Reply:
Also dot missing after Add -- conf